provider/aws.go

```diff
@@ -152,12 +152,18 @@ func (p *AWSProvider) Records() (endpoints []*endpoint.Endpoint, _ error) {
 				continue
 			}
 
+			var ttl endpoint.TTL
+			if r.TTL == nil {
+				ttl = endpoint.TTL(0)
```

Comment:
This should not be needed, because it's already the default: https://play.golang.org/p/KwP4sN91vx